voluncheer/opportunityboard/test_commands.py

```diff
@@ -38,3 +46,120 @@ def test_load_categories(self):
             self.assertEqual(len(Category.objects.all()), 1)
             self.assertEqual(len(Subcategory.objects.all()), 1)
             self.assertEqual(len(Subsubcategory.objects.all()), 1)
+
+
+class TestArchiveOpportunities(TestCase):
+    """Tests the archive opportunities command."""
+
+    def setUp(self):
+        self.org = Organization.objects.create(
+            user=User.objects.create(
+                email="jedi@jedi.com",
+                password="peace_and_justice_for_the_galaxy",
+                type=UserType.ORGANIZATION,
+            ),
+            name="Jedi Council",
+        )
+
+        self.now = datetime.now(pytz.utc)
+        description = "Please help us support our community at this week's soup kitchen"
+
+        self.opportunity1 = Opportunity.objects.create(
+            organization=self.org,
+            category=Category.objects.filter(name=_CATEGORY).first(),
+            title="Test opportunity 1",
+            description=description,
+            date=timezone.now() - timedelta(days=1),
```

Comment:
I suggest using fixed dates for the datetimes. Although the logic is correct as is, using fixed dates will ensure that the tests run consistently.

For example:
```
today = datetime(year=2023, month=4, day=16, tzinfo=dt.timezone.utc)
...
self.opportunity1 = Opportunity.objects.create( ...
date = today - timedelta(days=1)
```

Reply:
That said, this would only work if you made the above changes to the command so you could call it for a specific date.

Reply:
Out of curiosity, why would it be bad to use timezone for tests?

Reply:
Because it is non-deterministic and the test will never be exactly the same any time you run it. Say for example you used `datetime.now()` and asserted it was before a fixed date well in the future. Your test could be successful for years, but eventually when the fixed date passes, it would no longer pass. 

This isn't an issue in this case since you aren't mixing in fixed dates, but just an example to illustrate how the test is no longer locked in place when using `now()`.